src/components/StoreLink/StoreLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { pickBy } from 'lodash'
+import { withClient } from 'cozy-client'
+
+class DumbStoreLink extends React.Component {
+  static propTypes = {
+    type: PropTypes.oneOf(['konnector', 'webapp']),
+    category: PropTypes.string
+  }
+
+  redirect = async () => {
+    const { client: cozyClient, type, category } = this.props
+    const redirectionURL = await cozyClient.intents.getRedirectionURL(
+      'io.cozy.apps',
+      pickBy({ type, category }, Boolean)
+    )
```

Comment:
Nit: The getRedirectionURL could be done sooner than the click so that the user does not have to wait.